core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -3719,3 +3767,19 @@ class ReplicaManagerTest {
     }
   }
 }
+
+class MockReplicaSelector extends ReplicaSelector {
+
+  private val triggerSelectionCount = new AtomicLong()
+
+  def getTriggerSelectionCount: Long = triggerSelectionCount.get
+
+  /**
+   * Select the preferred replica a client should use for fetching. If no replica is available, this will return an
+   * empty optional.
+   */
```

Comment:
nit: I think that we can remove this default comment.

Reply:
@dajac  Thanks for your review. All comments have been addressed.